optuna/integration/__init__.py

```diff
@@ -64,7 +63,6 @@
     from optuna.integration.skorch import SkorchPruningCallback
     from optuna.integration.tensorboard import TensorBoardCallback
     from optuna.integration.tensorflow import TensorFlowPruningHook
-    from optuna.integration.tfkeras import TFKerasPruningCallback
```

Comment:
Ditto.

Reply:
Thanks. Fixed in https://github.com/optuna/optuna/pull/4662/commits/f8b651d6667205f24ac27788045398196ff6e615.